File: lib/models/supplier.dart
class Supplier {
  const Supplier({
    required this.id,
    required this.name,
    required this.latitude,
    required this.longitude,
    required this.expectedKg,
    required this.status,
    this.location = '',
    this.distance = '',
  });

  final int id;
  final String name;
  final double latitude;
  final double longitude;
  final double expectedKg;
  final String status;
  final String location;
  final String distance;
}

File: BackendAPI/WasteGlassAPI/Models/Supplier.cs
namespace WasteGlassAPI.Models;

public class Supplier
{
    public int Id { get; set; }
    public string SupplierId { get; set; } = string.Empty;
    public string Name { get; set; } = string.Empty;
    public string Location { get; set; } = string.Empty;
    public double Latitude { get; set; }
    public double Longitude { get; set; }
    public double ExpectedKg { get; set; }
    public string BarcodeValue { get; set; } = string.Empty;
    public string Status { get; set; } = string.Empty;
}
